src/css/base.scss

```diff
@@ -2282,3 +2282,7 @@ textarea::placeholder {
     left: 50%;
     transform: translate(-50%, -50%);
 }
+
```

Comment:
We already have this classname in base.scss

Reply:
done